base_utils.py

```diff
@@ -263,5 +263,6 @@ async def process_with_overlap(req, attribute_service, POST_PROMPT, Posts):
             )
             all_dict_meta+= result['post_meta']
         except Exception as e:
-            print(f'❌ Lỗi xử lý batch: {batch}\n{e}')
```

Comment:
High severity: Printing 'Production mode tuantuan' in production may expose environment details. Replace with a secure logging approach.\n```suggestion
import logging
logging.info("Application running in production mode")
```